src/win/timer.c

```diff
@@ -153,6 +153,12 @@ uint64_t uv_timer_get_repeat(const uv_timer_t* handle) {
 }
 
 
+uint64_t uv_timer_get_timeout(const uv_timer_t* handle) {
+  assert(handle->type == UV_TIMER);
+  return handle->timeout;
```

Comment:
master (which has @saghul timer refactoring uses timeout). v1.x on the other hand does not.